src/screens/OrganizationPeople/OrganizationPeople.tsx

```diff
@@ -84,6 +82,11 @@ function OrganizationPeople(): JSX.Element {
   })[1];
 
   useEffect(() => {
+    const filterData = {
+      firstName_contains: firstNameRef.current?.value ?? '',
+      lastName_contains: lastNameRef.current?.value ?? '',
+    };
+
     if (state === 0) {
```

Comment:
Could you move the logic that changes the filter object and executes the refetch to another object. I see the same logic is used elsewhere in the screen.

Reply:
Thanks for pointing that out, I've fixed that